README.md

```diff
@@ -562,6 +563,17 @@ interface BigNumber extends Node {
 
 **BigNumber** represents a big number.
 
+### `Audio`
+
+```ts
+interface Audio extends Node {
```

Comment:
Suggestion: Similarly to my PRs we should start documenting the fields with their purpose